frontend/src/components/form/base/ENumberField.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <ValidationProvider
+    v-slot="{ errors: veeErrors }"
+    ref="validationProvider"
+    tag="div"
+    :name="validationLabel"
+    :vid="veeId ?? path"
+    :rules="veeRules"
+    :required="required"
+    class="e-form-container"
+  >
+    <v-text-field
+      ref="textField"
+      v-bind="$attrs"
+      :filled="filled"
+      :required="required"
+      :hide-details="hideDetails"
+      :error-messages="veeErrors.concat(errorMessages)"
+      :label="labelOrEntityFieldLabel"
+      :class="[inputClass]"
+      v-on="inputListeners"
+    >
+      <!-- passing through all slots -->
+      <slot v-for="(_, name) in $slots" :slot="name" :name="name" />
+      <template v-for="(_, name) in $scopedSlots" :slot="name" slot-scope="slotData">
+        <slot :name="name" v-bind="slotData" />
+      </template>
+    </v-text-field>
+  </ValidationProvider>
+</template>
+
+<script>
+import { ValidationProvider } from 'vee-validate'
+import { formComponentPropsMixin } from '@/mixins/formComponentPropsMixin.js'
+import { formComponentMixin } from '@/mixins/formComponentMixin.js'
+
+export default {
+  name: 'ENumberField',
+  components: { ValidationProvider },
+  mixins: [formComponentPropsMixin, formComponentMixin],
+  computed: {
+    inputListeners: function () {
+      const vm = this
+      return Object.assign(
+        {},
+        // attach all $parent listeners
+        this.$listeners,
+        // override @input listener for correct handling of numeric values
+        {
+          input: function (value) {
+            if (/\d/.test(value) || /\./.test(value) || !/,/.test(value)) {
```

Comment:
When I paste "Hello, World?" into a number field then there's a PATCH request setting quantity to null (expected I guess) but the text stays as is. Only on page reload do you see that the quantity field is actually empty. Pasting text without a comma results in a blank number field.

Reply:
@simfeld you are right. The comma test is already in the line below and just needs a number test. Fixed the behavior to match the normal number field.